### DISCUSSION TITLE
Troubleshooting: CondaHTTPError: HTTP 000 CONNECTION FAILED

### DISCUSSION THREAD

**Discussion by ialbert** [biostars/biostar-handbook-issues#185] [General]
Many times HTTPS connection errors are caused by particular firewall settings,  Norton antivirus app, or wifi proxy settings. Eliminate these as factors, perhaps connect to a different provider or double-check the firewall settings. 

Most of the time the above solves the problem.

---

But there is a particular intermittent error that some users experience that manifests itself via the following error message:

    CondaHTTPError: HTTP 000 CONNECTION FAILED

The cause is somewhat mysterious but there are a few reported workaround:

The potential solution was to do a:

    find miniconda3/ -type f -exec touch {} +

once after installing conda and once after installing mamba.

References:

* https://github.com/biostars/biostar-handbook-issues/issues/157#issuecomment-1014141326
* https://github.com/conda/conda/issues/9948#issuecomment-909989810
* https://github.com/biostars/biostar-handbook-issues/issues/176

**Comment by njbowen** [comment]
The find ...touch command triggered the antivirus software on one student's machine and was stopping python 3.9 from running in the mamba install.  We set norton to allow python 3.9 always on the antivirus (norton) program 

Now the newest install script: curl http://data.biostarhandbook.com/install.sh | bash exits with this error: 

Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
##Installing bioinformatics software
xargs: mamba: No such file or directory

so we returned to step by step and the conda install mamba command fails with a python3.9 corruption error.
